src/table.ts

```diff
@@ -937,19 +958,30 @@ Please use the format 'prezzy' or '${instance.name}/tables/prezzy'.`);
             return;
           }
           if (
-            numRequestsMade <= maxRetries &&
+            numConsecutiveErrors < maxRetries &&
             RETRYABLE_STATUS_CODES.has(error.code)
           ) {
-            makeNewRequest();
+            const backOffSettings =
+              options.gaxOptions?.retry?.backoffSettings ||
+              DEFAULT_BACKOFF_SETTINGS;
+            const nextRetryDelay = getNextDelay(
+              numConsecutiveErrors,
+              backOffSettings
+            );
+            retryTimer = setTimeout(makeNewRequest, nextRetryDelay);
+            numConsecutiveErrors++;
```

Comment:
I think this should be moved to before the if statement. Probably the first thing in the error callback.  (and `numConsecutiveErrors <` changed to `numConsecutiveErrors <=`)

Reply:
I think it should still be numConsecutiveErrors < because we're not counting the first request we sent

Reply:
ok, please move it :)

Reply:
I wait I think i misunderstood your comment. Updated it